packages/launcher/src/main/rpc/app.js

```diff
@@ -213,7 +213,10 @@ export const sandboxed = {
 
   contacts_select: withPermission(
     'CONTACTS_READ',
-    async (ctx: AppContext, params: { multi?: boolean }) => {
+    async (
+      ctx: AppContext,
+      params: { multi?: boolean, options?: { withWallet?: boolean } },
```

Comment:
Rather than adding `options.withWallet`, I think `withWallet` could be set in the `params` object directly, such as `params: { multi?: boolean, withWallet?: boolean }`